pom.xml

```diff
@@ -185,5 +199,28 @@
             <artifactId>xmlsec</artifactId>
             <version>1.5.8</version>
         </dependency>
+
+        <!-- The following dependencies are not currently available from public repositories.
+             Unfortunately, Signicat does not currently maintain a Maven repository, therefore
+             users must manually download these jars from the Signicat website to be able to compile and run
+             this plugin within the Curity server.
+             A work-around to compile the plugin code without having access to the Signicat jars is to use
+             the following Maven command:
+
+               mvn -B -V -f external-pom.xml -P external clean verify
+
+             This will use the stubbed Signicat API provided in the ./java-connector/ directory to be able to
+             compile the source code of this plugin.
+             -->
+        <dependency>
+            <groupId>com.signicat</groupId>
+            <artifactId>java-connector</artifactId>
+            <version>4.0.1</version>
+        </dependency>
+        <dependency>
+            <groupId>com.signicat</groupId>
```

Comment:
this isn't needed. The OpenSAML types that are used are in that "mock" defined above `com.signicat:java-connector:4.0.1`